app/src/Pages/Home/index.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  getPublicData,
+  publicDataRequest,
+  allOffices,
+  availableCampaigns,
+  selectedOffices,
+  selectedCampaigns,
+  selectedStartDate,
+  selectedEndDate,
+  filteredPublicData,
+  campaignsTable,
+  mapData,
+  aggregatedContributorTypes,
+  aggregatedDonationSize,
+  aggregatedContributionsByRegion,
+  donationSizeByDonationRange,
+  setSelectedOffices,
+  setSelectedCampaigns,
+  setSelectedStartDate,
+  setSelectedEndDate,
+  summaryData,
+} from '../../state/ducks/publicData';
+import { showModal } from '../../state/ducks/modal';
+import Home from './Home';
+
+class HomePage extends React.Component {
```

Comment:
NBD, but I think the best practice here is to clarify what specifically you're using from the 'react' library at the top of the file like
```
import React, { Component } from 'react';
```

Reply:
Fixed the import. I'm going to keep with the class syntax here for now, but for CIVIC repo we can go full hooks.